tests/Microsoft.Identity.Test.Unit/CoreTests/OAuth2Tests/ClaimsTest.cs

```diff
@@ -251,6 +251,56 @@ public void ClaimsMerge_Test()
             Assert.AreEqual(TestConstants.ClientCapabilitiesAndClaimsJson, mergedJson);
         }
 
+        [TestMethod]
```

Comment:
Can these be merged into one data test method with params? The const names are descriptive enough.

Reply:
was focused on the coverage and missed reducing lines, good catch. but for whatever reason VS does not seem to like datatestmethod, think with the long input data something gets messed up and VS starts throwing exceptions

Reply:
tricky, updated